packages/core/components/ComboBox/index.tsx

```diff
@@ -67,7 +66,7 @@ export default function BaseComboBox(props: Props) {
             comboBoxOptionStyles={{
                 rootChecked: styles.comboBoxItemChecked,
             }}
-            useComboBoxAsMenuWidth={props?.useComboBoxAsMenuWidth}
+            useComboBoxAsMenuWidth={true}
```

Comment:
```suggestion
            useComboBoxAsMenuWidth
```

Reply:
This was deliberate; FluentUI was not registering the property correctly. I'm not able to replicate the bug now though, so will fix & keep an eye on it